ts/image-occlusion/store.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+import { writable } from "svelte/store";
+
+export const active = writable("");
+export const notesDataStore = writable({ id: "", title: "", divValue: "", textareaValue: "" }[0]);
```

Comment:
A docstring explaining what these values represent would be good. Same for active.